### DISCUSSION TITLE
Github account blocked

### DISCUSSION THREAD

**Discussion by MMar1ey13** [vercel/community#3508] [Help] [answered]
### Summary

It's my first time trying to deploy on vercel and when I try using Github to log in I'm told that my account is blocked. I even can't log in using my email. What could be the problem.
![WhatsApp Image 2023-08-04 at 11 35 29 AM](https://github.com/vercel/community/assets/127131158/a46ab25f-5a50-4237-af34-cfe2759a3792)
![WhatsApp Image 2023-08-04 at 12 13 37 PM](https://github.com/vercel/community/assets/127131158/a0c7fb9f-722f-4ebd-a3ca-5cddb0117fbf)
 

### Steps to Reproduce

I've provided screenshots above

**Comment by fabricio-fn** [comment]
did you receive any email from github informing the block or suspension?

**Reply by MMar1ey13** [reply]
No..They only sent the email to confirm authorization and a welcome email from Vercel

**Reply by fabricio-fn** [reply]
apparently you can use github smoothly, but I don't know why vercel blocks your account.

very strange, if vercel sent a welcome email it means you should have successfully logged in

**Reply by fabricio-fn** [reply]
try logging out and logging in again

**Comment by fabricio-fn** [comment]
Try clearing your browser's cookies and cache and try logging in again.

**Reply by MMar1ey13** [reply]
Didn't work

**Comment by amyegan** [comment, marked as answer]
If you're still unable to log in, please send an email to [registration@vercel.com](mailto:registration@vercel.com) with the email and username for the account.

**Reply by MMar1ey13** [reply]
I sent them an email and my issue was resolved. Thanks a lot
